packages/lit-html/src/lit-html.ts

```diff
@@ -1200,8 +1200,8 @@ export class AttributePart {
     element: HTMLElement,
     name: string,
     strings: ReadonlyArray<string>,
-    parent?: Disconnectable,
-    _options?: RenderOptions
+    parent: Disconnectable | undefined,
+    public options: RenderOptions | undefined
```

Comment:
I prefer not to use constructor field initializers - it makes the shape of the class less clear when reading.

Reply:
Changed.